tests/conftest.py

```diff
@@ -661,10 +661,18 @@ def example_holdout_result_dict(example_holdout_evaluator_dict, example_rmse_met
     }
 
 
-
 @pytest.fixture
 def sample_design_space_execution_dict(generic_entity):
     ret = generic_entity.copy()
+    ret.update(
+        {
+            "design_space_id": str(uuid.uuid4()),
+            "status": {
```

Comment:
It looks like, until now, we have always unpacked the `Status` response in the backend. We could consider making a class for this. But I figured I could solve it in the `_pre_build` method for now.